docs/source/models.rst

```diff
@@ -180,6 +184,10 @@ if necessary. Generally, only conversions that are guaranteed to be lossless are
 example, int -> long or int -> double conversions are ok, long -> double is not. If the types cannot
 be made compatible, MLflow will raise an error.
 
+For column-based signatures with datetime inputs, precision is ignored. For example, datetime values with
```

Comment:
I would maybe add this into its own section similarly to `Handling Integers With Missing Values below`. 

So we could have `Handling Date and Time`. And explain that in python, datetime has precision build into type but we ignore it for dataframe signatures, but not for tensor signatures.